datasets/fake_news_filipino/README.md

```diff
@@ -0,0 +1,151 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- crowdsourced
+languages:
+- tl
+licenses:
+- unknown
+multilinguality:
+- monolingual
+size_categories:
+- 1K<n<10K
+source_datasets:
+- original
+task_categories:
+- text-classification
+task_ids:
+- fact-checking
+---
+
+# Dataset Card for Fake News Filipino
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage: [Fake News Filipino homepage](https://github.com/jcblaisecruz02/Filipino-Text-Benchmarks)**
+- **Repository: [Fake News Filipino repository](https://github.com/jcblaisecruz02/Filipino-Text-Benchmarks)**
+- **Paper: [LREC 2020 paper](http://www.lrec-conf.org/proceedings/lrec2020/index.html)**
+- **Leaderboard:**
+- **Point of Contact:[Jan Christian Cruz](mailto:jan_christian_cruz@dlsu.edu.ph)**
+
+### Dataset Summary
+
+Low-Resource Fake News Detection Corpora in Filipino. The first of its kind. Contains 3,206 expertly-labeled news samples, half of which are real and half of which are fake.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+The dataset is primarily in Filipino, with the addition of some English words commonly used in Filipino vernacular.
+
+## Dataset Structure
+
+### Data Instances
+
+Sample data:
+{
+  "label": "0",
+  "article": "Sa 8-pahinang desisyon, pinaboran ng Sandiganbayan First Division ang petition for Writ of Preliminary Attachment/Garnishment na inihain ng prosekusyon laban sa mambabatas."
+}
```

Comment:
````suggestion
```
{
  "label": "0",
  "article": "Sa 8-pahinang desisyon, pinaboran ng Sandiganbayan First Division ang petition for Writ of Preliminary Attachment/Garnishment na inihain ng prosekusyon laban sa mambabatas."
}
```
````